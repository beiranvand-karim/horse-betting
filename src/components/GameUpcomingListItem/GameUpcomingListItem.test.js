/* global describe, it, jest, expect*/
import {shallow} from "enzyme"
import {GameUpcomingListItem} from "./"
import React from "react"
import renderer from "react-test-renderer"
import {MemoryRouter} from "react-router-dom"

describe('<GameUpcomingListItem/>', () => {
   const game = {
      "id": "V4_2019-04-04_6_6",
      "startTime": "2019-04-04T20:14:00",
   };

   it('should render without crashing', () => {
      shallow(<GameUpcomingListItem game={game}/>)
   });

   it('should matches the snapshot', () => {
      const tree = renderer.create(<MemoryRouter><GameUpcomingListItem game={game}/></MemoryRouter>).toJSON();
      expect(tree).toMatchSnapshot()
   });
});