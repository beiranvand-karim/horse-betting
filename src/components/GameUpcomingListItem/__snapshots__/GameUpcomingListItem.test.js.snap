// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GameUpcomingListItem/> should matches the snapshot 1`] = `
<div>
  <h4>
    id: 
    V4_2019-04-04_6_6
  </h4>
  <h5>
    start time: 
    2019-04-04T20:14:00
  </h5>
  <button
    onClick={[Function]}
  >
    +
  </button>
  <hr />
</div>
`;
